modules/infra/iam_users/eks_users.tf

```diff
@@ -0,0 +1,88 @@
+# make an iam user for each entry in the iam_eks_users list
+resource "aws_iam_user" "eks" {
+  for_each = toset(var.iam_eks_users)
+  name     = each.value
+}
+
+# give each user a temporary password for logging into aws web console
+resource "aws_iam_user_login_profile" "eks" {
+  for_each                = toset([for user in aws_iam_user.eks : user.name])
+  user                    = each.value
+  password_reset_required = true
+}
+
+resource "aws_iam_group" "eks" {
+  name = "eks"
+}
+
+# make each user a member of the EKS group
+resource "aws_iam_group_membership" "eks" {
+  name  = "eks"
+  users = [for user in aws_iam_user.eks : user.name]
+  group = aws_iam_group.eks.name
+}
+
+########################################
+# allow users to see everything
+
+data "aws_iam_policy" "ro" {
+  arn = "arn:aws:iam::aws:policy/ReadOnlyAccess"
```

Comment:
Does this mean people with the EKS role are able to see all the s3 files? AKA What level of readonly does this allow?

I think this is fine for now, but down the road I'd like to tighten up access to both the database and s3 files.

Reply:
It does! You can see everything this canned policy allows [here](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ReadOnlyAccess.html).

I've been kinda making up ACL policy as I go so definitely room for improvement. You thinking about the TF state bucket? If so we can put a policy on that bucket itself to exclude it from broad policies like this one.